V1Documentation.md

```diff
@@ -199,20 +163,20 @@ objects (for partial matches). Each should match the hostname or TLD that you wa
 An example:
 
 ```javascript
-rg4js('options', {
+Raygun.init('apikey', {
   allowInsecureSubmissions: true,
   ignoreAjaxAbort: true,
   ignoreAjaxError: true,
   debugMode: true,
   ignore3rdPartyErrors: false,
   wrapAsynchronousCallbacks: true,
-  excludedHostnames: ['\.local'],
-  excludedUserAgents: ['Mosaic'],
-  disableCrashReporting: false,
+  excludedHostnames: ['localhost', '\.dev'],
```

Comment:
Second should be '\.local' as before (should also be that in V2 readme.md if it's not)